app/controllers/agent/ManageAgentController.scala

```diff
@@ -313,7 +313,7 @@ class ManageAgentController @Inject()(
         params = GetPropertyLinksParameters(
           sortfield = ExternalPropertyLinkManagementSortField.ADDRESS,
           sortorder = ExternalPropertyLinkManagementSortOrder.ASC),
-        agentCode = agentCode,
-        backLink = controllers.agent.routes.ManageAgentController.manageAgent(Some(agentCode)).url
-      ))
+        agentCode = agentCode
```

Comment:
why did the backLink go?

Reply:
now we can only get to this particular page from one place so no need to pass it into the controller